packages/interface-adapter/test/quorum-getBlock.test.ts

```diff
@@ -20,9 +20,7 @@ async function prepareGanache(
     const server = Ganache.server({
       time: genesisBlockTime
     });
-    server.listen(port, (err: Error) => {
-      if (err) reject(err);
-
+    server.listen(port, () => {
```

Comment:
Why was this removed?  I'm not familiar with this code so it's not obvious to me.

Reply:
It looks like this typing changed in the latest minor release of `ganache-core`: https://github.com/trufflesuite/truffle/runs/951396343#step:6:371

Reply:
Oh weird.  Huh.  OK then!